workers/ohsome_quality_analyst/utils/helper.py

```diff
@@ -96,9 +96,9 @@ def update_features_indicator(feature, indicator):
     """adds the indicator metadata, result and resultdata (if available) to the
     properties of a feature"""
     if indicator.data is not None:
-        feature["properties"].update(vars(indicator.data))
-    feature["properties"].update(vars(indicator.metadata))
-    feature["properties"].update(vars(indicator.result))
+        feature["properties"].update(indicator.data)
+    feature["properties"].update(vars(indicator.metadata).copy())
+    feature["properties"].update(vars(indicator.result).copy())
```

Comment:
is it on purpose, that e.g. the `description` gets overwritten here?

Reply:
This change should be applied to the `update_features_report` function below as well.

Reply:
Those function will become obsolete in the next PR. Thats why I will not change it here.